operations/observability/mixins/workspace/dashboards/node-psi.json

```diff
@@ -531,7 +531,7 @@
         "name": "cluster",
         "options": [],
         "query": {
-          "query": "label_values(gitpod_ws_manager_workspace_phase_total, cluster)",
+          "query": "label_values(gitpod_ws_manager_mk2_workspace_phase_total, cluster)",
```

Comment:
Nice! I was just burned by this, when checking to see why a node was struggling. :pray: